tfhe/src/core_crypto/backward_compatibility/commons/math/random/mod.rs

```diff
@@ -19,6 +19,7 @@ pub enum DynamicDistributionVersions<T: UnsignedInteger> {
 }
 
 #[derive(Serialize)]
+#[cfg_attr(tfhe_lints, allow(tfhe_lints::serialize_without_versionize))]
```

Comment:
There is some limited support for external linting tool inside rustc. These tools should be able to use the `#[allow(tool_name::lint_name)]` syntax. However the syntax to declare a new tool is not stable yet. clippy can do that because it is hardcoded in rustc. bench and tarpaulin are not linting tool so they do not use the `#[allow]` syntax.
The trick is to run rustc with the unstable feature inside the tool and to gate this behind a `cfg_attr` so that it doesn't impact normal compilation. This is inspired by this: https://rust-marker.github.io/marker/book/usage/setting-lint-levels.html

Reply:
more info here: https://github.com/rust-lang/rust/issues/66079